core/build.gradle

```diff
@@ -33,11 +21,24 @@ compileGroovy {
     targetCompatibility = '1.6'
 }
 
+gradlePlugin {
+    plugins {
+        binrayRelease {
+            id = "com.novoda.bintray-release"
+            implementationClass = "com.novoda.gradle.release.ReleasePlugin"
+        }
+        legacy {
+            id = "bintray-release"
+            implementationClass = "com.novoda.gradle.release.ReleasePlugin"
+        }
+    }
+}
+
 publish {
     userOrg = 'novoda'
     groupId = 'com.novoda'
     artifactId = rootProject.name
-    version = project.version
+    version = "0.8.0"
```

Comment:
This is now a duplicate to https://github.com/novoda/bintray-release/blob/master/build.gradle#L8